templates/authorizationService-deployment.yaml

```diff
@@ -0,0 +1,119 @@
+    release: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.authorizationService.replicaCount }}
+  revisionHistoryLimit: {{ .Values.authorizationService.revisionHistoryLimit }}
+  selector:
+    matchLabels:
+      app: integration-manager
+      component: authorizationService
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+        checksum/sharedconfig: {{ include (print $.Template.BasePath "/shared-configmap.yaml") . | sha256sum }}
+        checksum/baseconfig: {{ include (print $.Template.BasePath "/authorizationService-configmap.yaml") . | sha256sum }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+        {{- if .Values.authorizationService.podAnnotations }}
+        {{ .Values.authorizationService.podAnnotations | toYaml | indent 8 | trim }}
+        {{- end }}
+      labels:
+        app: integration-manager
+        component: authorizationService
+        chart: {{ template "integration-manager.chart" . }}
+        release: {{ .Release.Name }}
+        {{- if .Values.authorizationService.extraLabels }}
+        {{ .Values.authorizationService.extraLabels | toYaml | indent 8 | trim }}
+        {{- end }}
+    spec:
+      {{- if .Values.authorizationService.affinity }}
+      affinity:
+        {{ .Values.authorizationService.affinity | toYaml | indent 8 | trim }}
+      {{- end }}
+      serviceAccountName: {{ template "integration-manager.fullname" . }}
+      {{- if .Values.authorizationService.nodeSelector }}
+      nodeSelector:
+        {{ .Values.authorizationService.nodeSelector | toYaml | indent 8 | trim }}
+      {{- end }}
+      {{- if .Values.authorizationService.extraInitContainers }}
+      initContainers:
+        {{ tpl .Values.authorizationService.extraInitContainers . | indent 6 | trim }}
+      {{- end }}
+      containers:
+      - name: datacloud-authorization-service
+        image: {{ .Values.authorizationService.image }}
+        imagePullPolicy: {{ .Values.authorizationService.pullPolicy }}
+        env:
+        {{- if .Values.authorizationService.javaOpts }}
+        - name: JAVA_OPTS
+          value: {{ .Values.authorizationService.javaOpts }}
+        {{- end }}
+        {{- if .Values.repository.awsCreateEnvVar }}
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              key: aws-access-key
+              {{- if .Values.existingSecret }}
+              name: {{ .Values.existingSecret }}
+              {{- else }}
+              name: {{ template "integration-manager.fullname" . }}
+              {{- end }}
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: aws-secret-key
+              {{- if .Values.existingSecret }}
+              name: {{ .Values.existingSecret }}
+              {{- else }}
+              name: {{ template "integration-manager.fullname" . }}
+              {{- end }}
+        {{- end }}
```

Comment:
I don't think you need the env block (lines 53-77)

Reply:
updated